src/main/java/org/powerflows/dmn/io/yaml/model/field/YamlFields.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2018-present PowerFlows.org - all rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.powerflows.dmn.io.yaml.model.field;
+
+import lombok.Data;
+
+import java.util.Map;
+
+@Data
+public final class YamlFields {
+    private Map<String, YamlInput> in;
```

Comment:
Maybe inputs and outputs to get plural?

Reply:
it will be to verbose imho... especialy needless in not exposed API